promql/engine.go

```diff
@@ -592,7 +592,7 @@ func (ng *Engine) populateSeries(ctx context.Context, q storage.Queryable, s *Ev
 		}
 		return nil
 	})
-	return querier, err, warnings
+	return querier, warnings, err
```

Comment:
Hmm, is your branch up to date? All these were already meant to have been fixed.